src/ReactTags.js

```diff
@@ -274,7 +274,12 @@ class ReactTags extends Component {
     e.preventDefault();
 
     const clipboardData = e.clipboardData || window.clipboardData;
-    const pastedText = clipboardData.getData('text');
+    let pastedText = clipboardData.getData('text');
+
+    // Checking if the pasted text is greater than maxLength props
+    if (this.props.maxLength && pastedText.length > this.props.maxLength) {
```

Comment:
can you destruct props  above
const { maxLength } = this.props;

Reply:
instead of writing this if can we do like this ?
```js
const maxTextLength = Math.min(maxLength, pastedText.length);
const pastedText = clipboardData.getData('text').substr(0, maxTextLength);
```

Reply:
can you also add a spec for this ?

Reply:
Thanks for the suggestions. I will try my best @ad1992